### ISSUE TITLE
Replace author to avoid duplicate links

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#444]
```diff
- - [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) by [@GitHubGuides](https://guides.github.com/)
+ - [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) by [@GitHub](https://github.com/)
```

Source to be changed https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/README.md#L96

**Comment by Gilnuss9** [comment]
not sure if I understood what you are asking.
do you want to replace the "@GitHubGuides" the just "GitHub"(while also changing what comes next)?

**Comment by erictleung** [comment]
@Gilnuss9 thanks for your interest! Yes, I'm suggesting to replace `@GitHubGuides` with `@GitHub` and to replace the corresponding link as well so that our continuous integration tool won't throw an error. I hope that makes sense.

**Comment by Gilnuss9** [comment]
> @Gilnuss9 thanks for your interest! Yes, I'm suggesting to replace `@GitHubGuides` with `@GitHub` and to replace the corresponding link as well so that our continuous integration tool won't throw an error. I hope that makes sense.

Understand, would like to be assigned to this

**Comment by Sky020** [comment]
I assume this is closed with #445 ?

**Comment by erictleung** [comment]
> 
> 
> I assume this is closed with #445 ?

yep!

Thanks, @Gilnuss9, for taking care of this issue! :+1:

Closing as resolved.
